WizardPanels/useFocusOnTransition.js

```diff
@@ -12,7 +12,7 @@ const transitionHandlers = {
 				if (spotlightId && !currentSpotlight) {
 					Spotlight.focus(spotlightId);
 				}
-			});
+			}, 40);
```

Comment:
Is this the only way we can do?

Reply:
We don't have a handler that is notified when all child components finish rendering.
I believe this is necessary to ensure that the rendering cycles are completed, allowing Spotlight to navigate to a target within the child components.